src/status_im/utils/money.cljs

```diff
@@ -225,8 +225,12 @@
 
 (defn format-members
   [count]
-  (if (> count 1000000)
+  (cond
+    (> count 1000000)
     (str (with-precision (/ count 1000000) 1) (i18n/label :t/M))
-    (if (and (> count 999) (< count 1000000))
-      (str (with-precision (/ count 1000) 1) (i18n/label :t/K))
-      count)))
+
+    (and (> count 999) (< count 1000000))
```

Comment:
@ilmotta looks better this way

Reply:
In case it helps, I often think about the `<` function like you can put it *between* the numbers, like what we do in math notation. I learned about this visualization trick from an old blog post. The same logic can be used for just 2 numbers using other comparison functions, for example `(> index 10)` could be represented in math as `index > 10`, so that's how I never forgot what these comparison functions do and how they work :)

```clojure
(< 999 count 1000000)

;; Conceptually the same as this in math notation:
;; 999 < count < 1000000
```